### PR TITLE
include more classes to generate the class hierarchical structure 

### PR DESCRIPTION
**Pull request by junchaoxia**
This PR is related to #2 for including more classes to generate the class hierarchical structure to propose the design of whole package. No detailed implementations are included.

### CODE REVIEW

File: src/aspyre/oreintation/__init__.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Oreint3D:
```
Comment (vineetbansal) [inline src/aspyre/oreintation/__init__.py]: 'Orientation' (misspelling) - needs to be replaced in filenames/foldernames as well as inside files.

File: src/aspyre/utils/blk_diag_func.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
+import numpy as np
```
Comment (vineetbansal) [inline src/aspyre/utils/blk_diag_func.py]: This file has severe pep8 issues and no tests. Looks like it's not used anywhere. Do you want to not include it in this PR and only submit it when it's done?

File: src/aspyre/denoise/covar2d_ctf.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
+import logging
```
Comment (vineetbansal) [inline src/aspyre/denoise/covar2d_ctf.py]: No tests? There's got to be some when you were writing all this code..

File: src/aspyre/oreintation/__init__.py (hunk 1 above)
Comment (junchaoxia) [reply src/aspyre/oreintation/__init__.py]: Good. Fixed in the new commit.

File: src/aspyre/utils/blk_diag_func.py (hunk 2 above)
Comment (junchaoxia) [reply src/aspyre/utils/blk_diag_func.py]: This file is for the operations of block diagonalized matrix for denoising module. Debuging and testing are not done.

File: src/aspyre/denoise/covar2d_ctf.py (hunk 3 above)
Comment (junchaoxia) [reply src/aspyre/denoise/covar2d_ctf.py]: This file is for denoising module. Debugging and testing are not done.  Only for generating the class structure for the whole package.

**Pull request merged**